src/biogeochem/VOCEmissionMod.F90

```diff
@@ -933,13 +890,28 @@ function get_gamma_T(t_veg240_in, t_veg24_in,t_veg_in, ct1_in, ct2_in, betaT_in,
     if ( (t_veg240_in > 0.0_r8) .and. (t_veg240_in < 1.e30_r8) ) then 
        ! topt and Eopt from eq 8 and 9:
        topt = co1 + (co2 * (t_veg240_in-tstd0))
-       Eopt = Ceo_in * exp (co4 * (t_veg24_in-tstd0)) * exp(co4 * (t_veg240_in -tstd0))
+       if ( (ivt_in == nbrdlf_dcd_brl_shrub) ) then  ! boreal-deciduous-shrub
+       ! coming from BEAR-oNS campaign willows results
+             Eopt = 7.9_r8 * exp (0.217_r8 * (t_veg24_in - tfrz - 24.0_r8))
```

Comment:
It would be good coding practice to give names to 7.9 and 0.217. 

I see that eq. 2 (in your introduction to the PR) has no names for these but maybe you have some idea of what they represent. Also the second one does not match the exact value in the intro to the PR. Does this concern you?

Reply:
7.9 and 0.217 are empirical parameters for the emission factors of boreal deciduous broadleaf shrubs in response to 1-day temperature change. The second number, 0.217, does not concern me from here.